pysrc/hdmi_switcher_server.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+import os
+import sys
+import json
+import serial
+from flask import Flask, request
+import logging
+from logging.config import dictConfig
+
+config_file = os.path.join(os.path.dirname(__file__),
+                '../config/serial-settings.json')
+
+if not os.path.isfile(config_file):
+    raise RuntimeError("Config file {0} not found".format(config_file))
+
+with open(config_file) as f:
+    config = json.load(f)
+    print('Loaded config file {0}'.format(config_file))
+
+dictConfig({
+    'version': 1,
+    'formatters': {'default': {
+        'format': '[%(asctime)s] %(levelname)-7s %(module)-22s: %(message)s',
+    }},
+    'handlers': {'wsgi': {
+        'class': 'logging.StreamHandler',
+        'stream': 'ext://flask.logging.wsgi_errors_stream',
+        'formatter': 'default'
+    }},
+    'root': {
+        'level': config['server']['logging_level'],
+        'handlers': ['wsgi']
+    }
+})
+
+app = Flask(__name__)
+
+
+@app.route('/', methods=['POST'])
+def handle_serial():
+    request_json = request.get_json()
+    app.logger.debug('Request JSON {0}'.format(request_json))
+
+    with open(config_file) as f:
+        config = json.load(f)
```

Comment:
Looks like this is already loaded into the global scope up above (L17)

Reply:
I kept this here so that the hdmi map is dynamically loaded upon each request, so the server doesn't need to be taken down if the mapping changes

Reply:
👍🏾